src/crc32.rs

```diff
@@ -31,6 +31,7 @@ impl<R> Crc32Reader<R> {
         self.check == self.hasher.clone().finalize()
     }
 
+    #[allow(dead_code)]
```

Comment:
Just delete this method if it's no longer used.

Reply:
Done!